package.json

```diff
@@ -48,7 +48,7 @@
     "localforage": "1.10.0",
     "localforage-observable": "2.1.1",
     "lodash": "4.17.21",
-    "luxon": "2.4.0",
```

Comment:
@hmhealey Luxon is exposed in `window` in plugins export and is also marked as shared in module federation. Would we need to notify plugin creators and/or other teams internally if we want to move forward with the upgrade?

Reply:
Plugin exports are more the concern there since module federation actually lets containers (in our case, other products) declare what version of a package they expect, and if they use an incompatible one for a non-singleton package, they can actually have the app load two different versions, so if there was a breaking change, they could just load their own version of Luxon as well.

That said, I think this is fine. The breaking change for 3.0 is minor, and I doubt the ESM imports would change anything since the plugin template runs everything through Babel

Reply:
Thanks a lot @hmhealey ! cc: @amyblais just for visibility but probably no action here if we don't need to notify anyone